src/reynir/Greynir.grammar

```diff
@@ -2284,16 +2296,30 @@ Sögn_2_nh →
 
 Andlög/fall/afall →
     NlÓbeintAndlag/fall AndlögBotn/fall/afall
+    | UmsnúinAndlög/fall/afall
 
 Andlög/fall/afall/tala/kyn →
     NlÓbeintAndlag/fall LoTengtSögn_nf/tala/kyn? AndlögBotn/fall/afall
+    | UmsnúinAndlög/fall/afall/tala/kyn
```

Comment:
Since /tala/kyn is never used in `UmsnúinAndlög`, it can (and should) be omitted here